build.sh

```diff
@@ -284,6 +284,12 @@ fi
 # Downloading
 ######################################
 
+kill_object_server() {
+    (pgrep -f realm-object-server || true) | while read pid; do
```

Comment:
This won't kill `build.sh start-object-server` or `./sync/start-object-server.command`. Thoughts on changing this to `pgrep -f object-server`?